Add Cortex doc (#1595)
SEKOIA-IO/documentation: docs/xdr/features/collect/integrations/endpoint/paloalto_cortex_edr.md

uuid: b5968b0a-1206-4d3b-b960-e6b61e7f6ee6

> name: Palo Alto Cortex XDR

Reviewer: ```suggestion
name: Palo Alto Cortex EDR
```

type: intake

## Overview

> Palo Alto Networks Cortex XDR is a comprehensive cybersecurity platform designed to provide advanced threat detection, prevention, and response capabilities. Cortex XDR goes beyond traditional endpoint protection by integrating data from various security sources, including endpoints, networks, and cloud environments, to deliver a holistic view of the organization's security landscape. 

Reviewer: ```suggestion
Palo Alto Cortex EDR is an advanced Endpoint Detection and Response solution offering real-time threat detection, investigation, and response capabilities, empowering organizations to proactively defend against sophisticated cyber threats across their endpoints.
```

> This setup guide shows how to forward events collected on the Cortex platform to Sekoia.io.

Reviewer: ```suggestion
This setup guide shows how to forward EDR events collected on the Palo Alto Cortex XDR platform to Sekoia.io.
```

{!_shared_content/operations_center/detection/generated/suggested_rules_9b95c9cf-8b78-4830-a1ed-b9e88f05e67a_do_not_edit_manually.md!}

{!_shared_content/operations_center/integrations/generated/9b95c9cf-8b78-4830-a1ed-b9e88f05e67a.md!}

## Create API Key

Before using the **Cortex XDR** connector, you must generate an API key in Cortex XDR. For that you need to follow steps bellow :

1. Within **Cortex XDR**, click **Settings**.
2. Click **Configurations**.

    ![Configuration Page](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/configuration.png){: style="max-width:100%; height: 500px;"}

3. In the **Configurations** menu, select **API Keys**, which appears in the **Integrations** section.
4. In the API Keys window, click **+New Key** button to create a new API key. And the **Generate API Key** window will open.

    ![Api Key Page](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/api_key.png){: style="max-width:100%"}

5. Select the **Standard** Security Level.

> 6. Select a role for the API key. After selecting the role, click **Generate**.

Reviewer: Which role the customer should assign to the API key?

Author: Yes they put that we need `Endpoint Management` permission. But as I don't have access to see the api key part in my platform I don't know if it's exactly the same name.

    ![Generation Button](/assets/operation_center/integration_catalog/cloud_and_saas/paloalto_cortex_edr/generate.png){: style="max-width:100%"}

7. After you’ve generated the key, copy the **API key** value presented and keep it secure.
8. Return to the **API Keys** window, and select the API key you just generated. Also make note of the **API key ID**, which appears in the API Keys table.

    !!! info
        - You will need also the `Fqdn` which is a unique host and domain name associated with each tenant. When you generate the API Key and Key ID, you are assigned an individual `Fqdn`.

## Configure

### Create the intake

> 1. Go to the [Intake page](https://app.sekoia.io/operations/intakes) and create a new `Palo Alto Cortex EDR` intake.

Reviewer: ```suggestion
1. Go to the [Intake page](https://app.sekoia.io/operations/intakes) and create a new `Palo Alto Cortex XDR` intake.
```
